map/framework.cpp

```diff
@@ -433,17 +433,17 @@ Framework::Framework(FrameworkParams const & params)
 
   m_bmManager = make_unique<BookmarkManager>(BookmarkManager::Callbacks(
     [this]() -> StringsBundle const & { return m_stringsBundle; },
-    [](std::vector<std::pair<df::MarkID, BookmarkData>> const & marks)
+    [this](std::vector<std::pair<df::MarkID, BookmarkData>> const & marks)
```

Comment:
is it possible to get rid of std:: here?

Reply:
Sure.